index.html

```diff
@@ -10,12 +10,16 @@
     <body>
         <form id="todoInput">
             <label for="inputField">Add ToDo:</label>
-            <input id="inputField" name="inputField" type="text">
-            <button type="submit">+</button>
+            <input id="inputField" name="inputField" type="text" value="">
+            <button  id="addBtn" type="submit">+</button><br>
+			<button  id="remAllBtn" type="submit">remove all</button>
```

Comment:
Please correct the intent - should be on the same line as the other buttons

```
 <button  id="addBtn" type="submit">+</button><br>
			<button  id="remAllBtn" type="submit">remove all</button>
```
===>
```
 <button  id="addBtn" type="submit">+</button>
 <br>
 <button  id="remAllBtn" type="submit">remove all</button>
```